apps/ideas/templates/advocate_europe_ideas/idea_detail.html

```diff
@@ -61,7 +64,7 @@
         </div>
         <div class="container-medium ideadetail-comments">
             <h4>{% trans "Write comment" %}</h4>
-            {% react_comments ideasketch %}
+            {% react_comments idea %}
```

Comment:
I kind of forgot what was the desired solution here? Should they share the comments or not?